src/platform/mbed/ConnectivityManagerImpl.cpp

```diff
@@ -34,432 +34,417 @@
 #include <type_traits>
 
 #if !CHIP_DEVICE_CONFIG_ENABLE_WIFI_STATION
-//#error "WiFi Station support must be enabled when building for mbed"
-#endif
-
-#if !CHIP_DEVICE_CONFIG_ENABLE_WIFI_AP
-//#error "WiFi AP support must be enabled when building for mbed"
+#error "WiFi Station support must be enabled when building for mbed"
 #endif
 
 using namespace ::chip;
 using namespace ::chip::Inet;
 using namespace ::chip::System;
 using namespace ::chip::DeviceLayer::Internal;
+
 namespace chip {
 namespace DeviceLayer {
 
 ConnectivityManagerImpl ConnectivityManagerImpl::sInstance;
 
-ConnectivityManager::WiFiStationMode ConnectivityManagerImpl::_GetWiFiStationMode(void)
+struct WorkData
```

Comment:
What is the purpose of this structure ?

Reply:
Oh, it is unnecessary residue;) I remove it